src/diffusers/schedulers/scheduling_edm_euler.py

```diff
@@ -84,15 +89,19 @@ def __init__(
         sigma_min: float = 0.002,
         sigma_max: float = 80.0,
         sigma_data: float = 0.5,
+        sigma_schedule: str = "karras",
         num_train_timesteps: int = 1000,
         prediction_type: str = "epsilon",
         rho: float = 7.0,
     ):
+        if sigma_schedule not in ["karras", "linear"]:
```

Comment:
I'm not sure about the names, I don't think it should be called linear, it is not linear, no? but not sure if we can call it cosine-continuous

Reply:
let's call it "exponential" here to be consistent with k-diffusion 
https://github.com/crowsonkb/k-diffusion/blob/6ab5146d4a5ef63901326489f31f1d8e7dd36b48/k_diffusion/sampling.py#L26